terraform/role-ci.tf

```diff
@@ -0,0 +1,101 @@
+# Example of how to use policies from terraform-aws-serverless to create
```

Comment:
Let's add a comment at the top here with link to FormidableLabs/terraform-aws-serverless#53 stating we're looking to move this into the module...